docs/code/demos/intermediate.py

```diff
@@ -7,33 +7,31 @@
 @app.cond()
 def is_foo():
     "This is a custom condition"
-    ...
     return True
 
 @app.task(daily & is_foo)
 def do_daily():
     "This task runs once a day when foo is true"
-    ...
```

Comment:
I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).